<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddReg</class>
 <widget class="QWidget" name="AddReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>503</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>添加规则</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>回复内容</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>431</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;</string>
    </property>
    <column>
     <property name="text">
      <string>回复内容</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>正则匹配</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>是否为文件</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>是否@</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="reply_add_pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加一行规则</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reply_del_pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除选中行</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>回复内容：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>正则匹配：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reply_is_file_type_checkBox">
    <property name="geometry">
     <rect>
      <x>452</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>是否为文件类型</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reply_is_at_checkBox">
    <property name="geometry">
     <rect>
      <x>582</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>是否@</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="reply_textEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="regex_textEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>721</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>新成员回复内容</string>
   </property>
   <widget class="QTableWidget" name="new_member_tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>431</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;</string>
    </property>
    <column>
     <property name="text">
      <string>回复内容</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>是否为文件</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>是否@</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="new_member_add_pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加一行回复</string>
    </property>
   </widget>
   <widget class="QPushButton" name="new_member_del_pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除选中行</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>回复内容：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="new_member_is_file_type_checkBox">
    <property name="geometry">
     <rect>
      <x>452</x>
      <y>120</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>是否为文件类型</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="new_member_is_at_checkBox">
    <property name="geometry">
     <rect>
      <x>582</x>
      <y>120</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>是否@</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="new_member_textEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>450</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>对于本页面修改要点击同步按钮才会保存到本地</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sync_file_pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>447</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>同步本页面所有规则</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
